ddd-koans/Value Object & Entity/Draft/src/Length.kt

```diff
@@ -0,0 +1,18 @@
+import java.math.BigDecimal
+import java.time.Duration
+
+data class Length(private val durationInSeconds: BigDecimal) {
+    operator fun plus(other: Length) = Length(durationInSeconds.plus(other.durationInSeconds))
+
+    val duration: Duration = Duration.ofMillis(durationInSeconds.multiply(1000.toBigDecimal()).toLong())
+}
+
+interface Lengthy {
```

Comment:
`Lengthy` doesn't sound good as it's adjective describing something too long, what about `LengthIn` or `LengthInTempo` ?